contracts/src/TeleporterTokenSource.sol

```diff
@@ -118,10 +118,16 @@ abstract contract TeleporterTokenSource is
             })
         );
 
-        emit TokensSent(messageID, msg.sender, input, amount);
+        if (isMultihop) {
+            emit TokensRouted(messageID, input, amount);
+        } else {
+            emit TokensSent(messageID, msg.sender, input, amount);
+        }
```

Comment:
I was leaning towards leaving it out, because `TokensSent` and `TokensAndCallSent` are only emitted on the chain the transfer is initiated from, and the blockchain ID in that case is always the chain being indexed. 

`TokensRouted` events I just thoguht to keep as minimal as possible. Thoughts?